WooCommerce/Classes/ViewRelated/JetpackSetup/JetpackSetupCoordinator.swift

```diff
@@ -45,37 +45,59 @@ final class JetpackSetupCoordinator {
                 self.presentJCPJetpackInstallFlow()
             }
         }, dismissAction: { [weak self] in
-            self?.navigationController.dismiss(animated: true, completion: nil)
+            self?.rootViewController.dismiss(animated: true, completion: nil)
         })
-        navigationController.present(benefitsController, animated: true, completion: nil)
+        rootViewController.present(benefitsController, animated: true, completion: nil)
         self.benefitsController = benefitsController
     }
+
+    func handleAuthenticationUrl(_ url: URL) -> Bool {
```

Comment:
@jaclync I added tests in https://github.com/woocommerce/woocommerce-ios/commit/2a3bf67ef5f4a04ccbd7de12c1ce09b439c31d5c, but the tests failed since we are keeping the scheme in the credentials file and I'm not sure if exposing the scheme is a security threat. So, I removed the tests - what would you suggest for this situation?

Reply:
I don't think the value is confidential since anyone can get this value by creating a WPCOM account and requesting a magic/auth link then following the URL. From the [Apple doc](https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app) on custom URL schemes, the important security measure is to "validate all URL parameters and discard any malformed URLs" as anyone can send and open a URL of the same scheme. Having unit tests on malformed URLs feels even more valuable in this case.

If you think it's better not to test the production URL scheme, we can still keep the other test cases.

Reply:
> since anyone can get this value by creating a WPCOM account and requesting a magic/auth link then following the URL

This is a valid point. I restore the tests with the app scheme in bd165bbb4046491ed67c1d2bcb3600b8fe214e52.